code/datums/emergency_calls/emergency_call.dm

```diff
@@ -164,7 +167,7 @@
 					if(!length(candidates)) //We ran out of candidates.
 						break
 					var/datum/mind/M = pick(candidates) //Get a random candidate, then remove it from the candidates list.
-					if(M.current.stat != DEAD)
+					if(M?.current?.stat != DEAD)
 						candidates -= M //Strip them from the list, they aren't dead anymore.
 						continue
 					if(!istype(M)) //Something went horrifically wrong
```

Comment:
this check should be above the previous check

Reply:
Done.